services/web/client/source/class/osparc/auth/core/Utils.js

```diff
@@ -20,6 +20,14 @@ qx.Class.define("osparc.auth.core.Utils", {
 
   statics:
   {
+    passwordLengthValidator: function(value, item) {
+      const valid = value != null && value.length > 3;
```

Comment:
the length and complexity of the passwords shall be increased in the near future before releasing

Reply:
@odeimaiz I found this

Password Requirements

- Between 8-50 characters
- At least 1 upper and lower case letter
- At least 1 number
and further reading https://medium.com/tradecraft/improving-your-passwords-65270dd877b